docs/op-guide/deploy-tikv-using-docker-compose.md

```diff
@@ -20,63 +20,143 @@ Make sure you have installed the following items on your machine:
+## Install 
 
-1. Download `tidb-docker-compose`.
+Download `tidb-docker-compose`.
 
-    ```bash
-    git clone https://github.com/pingcap/tidb-docker-compose.git
+```bash
+git clone https://github.com/pingcap/tidb-docker-compose.git
+```
+
+## Prepare cluster
+
+Here we will run a simple cluster with only 1 PD and 1 TiKV. The base docker compose config is:
+
+```yaml
+version: '2.1'
+
+services:
+  pd0:
+    image: pingcap/pd:latest
+    ports:
+      - "2379"
+    volumes:
+      - ./config/pd.toml:/pd.toml:ro
+      - ./data:/data
+      - ./logs:/logs
+    command:
+      - --name=pd0
+      - --client-urls=http://0.0.0.0:2379
+      - --peer-urls=http://0.0.0.0:2380
+      - --advertise-client-urls=http://pd0:2379
+      - --advertise-peer-urls=http://pd0:2380
+      - --initial-cluster=pd0=http://pd0:2380
+      - --data-dir=/data/pd0
+      - --config=/pd.toml
+      - --log-file=/logs/pd0.log
+    restart: on-failure
+  
+  tikv0:
+    image: pingcap/tikv:latest
+    volumes:
+      - ./config/tikv.toml:/tikv.toml:ro
+      - ./data:/data
+      - ./logs:/logs
+    command:
+      - --addr=0.0.0.0:20160
+      - --advertise-addr=tikv0:20160
+      - --data-dir=/data/tikv0
+      - --pd=pd0:2379
+      - --config=/tikv.toml
+      - --log-file=/logs/tikv0.log
+    depends_on:
+      - "pd0"
+    restart: on-failure
+```
+
+All the following example docker compose config file will contain this base config.
+
+## Example
+
+### [YCSB](https://github.com/pingcap/go-ycsb)
+
+1. Create a `ycsb-docker-compose.yml` file, add the above base config to this file, and then append the following section:
+
+    ```yaml
+    ycsb:
+      image: pingcap/go-ycsb
     ```
 
-2. Edit the `compose/values.yaml` file to configure `networkMode` to `host`.
+2. Start the cluster:
 
     ```bash
-    cd tidb-docker-compose
-    vim compose/values.yaml  
+    rm -rf data logs 
+    docker-compose -f ycsb-docker-compose.yml pull 
+    docker-compose -f ycsb-docker-compose.yml up -d
     ```
 
-3. Edit the `compose/values.yaml` file to comment the TiDB section out.
-
-4. Change the Prometheus and Pushgateway addresses for the `host` network mode.
+3. Start YCSB:
 
     ```bash
-    sed -i 's/pushgateway:9091/127.0.0.1:9091/g' config/*
-    sed -i 's/prometheus:9090/127.0.0.1:9090/g' config/*
+    docker-compose -f ycsb-docker-compose.yml run ycsb shell tikv -p tikv.pd=pd0:2379
     ```
 
-5. Generate the `generated-docker-compose.yml` file.
+4. Run YCSB:
 
     ```bash
-    helm template compose > generated-docker-compose.yml
+    INFO[0000] [pd] create pd client with endpoints [pd0:2379]
+    INFO[0000] [pd] leader switches to: http://pd0:2379, previous:
+    INFO[0000] [pd] init cluster id 6628733331417096653
+    » read a
+    Read empty for a
+    » insert a field0=0
+    Insert a ok
+    » read a
+    Read a ok
+    field0="0"
     ```
 
-6. Create and start the cluster using the `generated-docker-compose.yml` file.
+### [Titan](https://github.com/meitu/titan)
```

Comment:
-> Use [Titan](https://github.com/meitu/titan) to connect to the TiKV cluster via the Redis protocal: